src/types/configData.go

```diff
@@ -97,12 +97,14 @@ type ConfigClusterMgmt struct {
 }
 
 type ConfigObservability struct {
-	Enable              bool   `json:"enable,omitempty" bson:"enable,omitempty"`
-	CronJobTimeInterval string `json:"cronjob_time_interval,omitempty" bson:"cronjob_time_interval,omitempty"`
-	DBName              string `json:"db_name,omitempty" bson:"db_name,omitempty"`
-	SysObservability    bool   `json:"sys_observability,omitempty" bson:"sys_observability,omitempty"`
-	NetObservability    bool   `json:"net_observability,omitempty" bson:"net_observability,omitempty"`
-	WriteLogsToDB       bool   `json:"write_logs_to_db,omitempty" bson:"write_logs_to_db,omitempty"`
+	Enable                bool   `json:"enable,omitempty" bson:"enable,omitempty"`
+	CronJobTimeInterval   string `json:"cronjob_time_interval,omitempty" bson:"cronjob_time_interval,omitempty"`
+	DBName                string `json:"db_name,omitempty" bson:"db_name,omitempty"`
+	SysObservability      bool   `json:"sys_observability,omitempty" bson:"sys_observability,omitempty"`
+	NetObservability      bool   `json:"net_observability,omitempty" bson:"net_observability,omitempty"`
+	WriteLogsToDB         bool   `json:"write_logs_to_db,omitempty" bson:"write_logs_to_db,omitempty"`
+	CronJobTimeForPurgeDB string `json:"cronjob_time_for_purge_db,omitempty" bson:"cronjob_time_for_purge_db,omitempty"`
+	PurgeLogsFromDB       bool   `json:"purge_logs_from_db,omitempty" bson:"purge_logs_from_db,omitempty"`
```

Comment:
```suggestion
	PurgeOldDBEntries       bool   `json:"purge_logs_from_db,omitempty" bson:"purge_logs_from_db,omitempty"`
```

Reply:
Accepted and Thank you for suggesting and reviewing.